stripe/test/resources/test_customers.py

```diff
@@ -234,20 +234,38 @@ def test_legacy_delete_subscription(self):
             None
         )
 
+    def test_list_customer_subscriptions(self):
+        customer = stripe.Customer.construct_from({
+            'id': 'cus_foo',
+            'subscriptions': {
+                'object': 'list',
+                'url': '/v1/subscriptions',
```

Comment:
Actually, the old `customer.subscriptions` methods also call the new URL - this is because the URL is set on the subscription object, which is the same class whether through the customer or by itself. This is why we could not roll this out until all the data had been verified. Otherwise it would have been fine to roll out if only the new methods called the new endpoints.